frontend/packages/data-portal/app/types/state.ts

```diff
@@ -0,0 +1,8 @@
+/**
+ * Type definitions exported from useHydrateAtoms.d.ts
+ */
+
+import { WritableAtom } from 'jotai'
+
+export type AnyWritableAtom = WritableAtom<unknown, any[], any>
+export type AtomTuple<A = AnyWritableAtom, V = unknown> = readonly [A, V]
```

Comment:
nit - rename this so that it says that it's the atom with the value or something?